src/frontend/packages/core/tsconfig.app.json

```diff
@@ -3,11 +3,11 @@
   "compilerOptions": {
     "outDir": "../../../../out-tsc"
   },
-  "exclude": [
-    "./src/test.ts",
-    "**/*.spec.ts",
-    "**/*.spec.helpers.ts",
-    "**/node_modules/**",
-    "./test-framework/**"
+  "files": [
```

Comment:
Thought this broke cloud-foundry code splitting, however replaced with the old version and it stills like it's included in core